awx/main/models/workflow.py

```diff
@@ -366,6 +366,32 @@ class WorkflowJobOptions(LaunchTimeConfigBase):
     class Meta:
         abstract = True
 
+    ask_inventory_on_launch = AskForField(
+        blank=True,
+        default=False,
+    )
+    ask_limit_on_launch = AskForField(
+        blank=True,
+        default=False,
+    )
+    ask_scm_branch_on_launch = AskForField(
+        blank=True,
+        default=False,
+    )
+    ask_labels_on_launch = AskForField(
+        blank=True,
+        default=False,
+    )
+    ask_tags_on_launch = AskForField(
+        blank=True,
+        default=False,
+        allows_field='job_tags',
+    )
+    ask_skip_tags_on_launch = AskForField(
+        blank=True,
+        default=False,
+    )
```

Comment:
I don't see any reason we can't move these to the `SurveyJobTemplateMixin` abstract model, which is specifically intended for reuse between JTs and WFJTs. These aren't new fields, JTs already have them, so instead of copying them, can't we just move them up the class inheritance structure?